Elastic6/was-kibana.json

```diff
@@ -1,12 +1,12 @@
 [
   {
-    "_id": "Was-Problems-K5-20181221",
+    "_id": "Was-Problems-K5-20191129",
     "_type": "dashboard",
     "_source": {
-      "title": "WAS-Problems-K5-20181221",
+      "title": "WAS-Problems-K5-20191129",
```

Comment:
name should include K6 since this is the Kibana 6 dashboard.  Also, for Elastic5 file you made id be WAS-... instead of Was-...